src/Compilers/VisualBasic/Portable/Symbols/PropertySignatureComparer.vb

```diff
@@ -191,18 +200,19 @@ Done:
         End Function
 #End Region
 
-        Private Shared Function HaveSameTypes(prop1 As PropertySymbol, prop2 As PropertySymbol, considerCustomModifiers As Boolean) As Boolean
+        Private Shared Function HaveSameTypes(prop1 As PropertySymbol, prop2 As PropertySymbol, comparison As TypeCompareKind) As Boolean
             If prop1.ReturnsByRef <> prop2.ReturnsByRef Then
                 Return False
             End If
 
             Dim type1 = prop1.Type
             Dim type2 = prop2.Type
 
-            ' the runtime compares custom modifiers using (effectively) SequenceEqual
```

Comment:
>' the runtime compares custom modifiers using (effectively) SequenceEqual [](start = 12, length = 73)

Not sure why the comment was removed. I believe there was another place where similar comment was removed.